lib/jwe/decrypt.js

```diff
@@ -285,17 +285,17 @@ function JWEDecrypter(ks, globalOpts) {
       promise = promise.then(function(jwe) {
         if ("DEF" === jwe.header.zip) {
           return new Promise(function(resolve, reject) {
-            zlib.inflateRaw(Buffer.from(jwe.plaintext), function(err, data) {
-              if (err) {
-                reject(err);
-              }
-              else {
-                jwe.payload = jwe.plaintext = data;
-                resolve(jwe);
-              }
-            });
+            try {
+              var data = pako.inflateRaw(Buffer.from(jwe.plaintext))
+
+              jwe.payload = jwe.plaintext = data;
```

Comment:
I think wrapping `data` with `Buffer.from` will get tests passing.  The API described in the README says this is supposed to be a node.js `Buffer`; pretty sure that is why tests are still failing.